ThunderCloud/TSCAccordionTabBarViewController.m

```diff
@@ -52,7 +52,7 @@ - (instancetype)initWithDictionary:(NSDictionary *)dictionary parentObject:(id)p
                 [typeDictionary setValue:@"NavigationTabBarViewController" forKey:@"type"];
                 
                 TSCNavigationTabBarViewController *navTabController = [[TSCNavigationTabBarViewController alloc] initWithDictionary:typeDictionary];
-                navTabController.title = TSCLanguageDictionary(tabBarItemDictionary[@"tabBarItem"][@"title"]);
+                navTabController.title = [[TSCStormLanguageController sharedController] stringForDictionary:(tabBarItemDictionary[@"tabBarItem"][@"title"])];
```

Comment:
Could we add some type safety in here whilst we're changing up the language controller? :)